include/GafferImage/OpenColorIOTransform.h

```diff
@@ -63,6 +63,7 @@ class GAFFERIMAGE_API OpenColorIOTransform : public ColorProcessor
 
 		/// May return null if the derived class does not
 		/// request OCIO context variable support.
+		/// \deprecated Use the OpenColorIOTransform node instead.
```

Comment:
I assume this should be "OpenColorIOContext node" instead of "OpenColorIOTransform node"?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5324/commits/f0bef0605b2f1b6788a6083e4ab1731fbb918efb.